src/api/models/bedrock.py

```diff
@@ -133,14 +133,32 @@ def list_models(self) -> list[str]:
         """Always refresh the latest model list"""
         global bedrock_model_list
         bedrock_model_list = list_bedrock_models()
+        if DEBUG:
+            logger.info("Bedrock model list: " + json.dumps(bedrock_model_list))
```

Comment:
If we're going to use the `DEBUG` env var, then we should print at the `debug` log level.
```suggestion
            logger.debug("Bedrock model list: " + json.dumps(bedrock_model_list))
```